OscLib/Makefile

```diff
@@ -4,16 +4,16 @@ SHELL = bash
 # Extremely crude stop-gap makefile
 
 # Record what flags were used. Can dump with
-# readelf -p .GCC.command.line libOscLib.so
+# readelf -p .GCC.command.line libOscLib.$(DSOEXT)
 CFLAGS := -std=c++17 -I.. -I${ROOT_INC} -I${EIGEN_INC} -I${BOOST_INC} -I${GSL_INC} -g -Wall -Wpedantic -Wextra -Werror -grecord-gcc-switches
 
 CFLAGS += $(shell if [[ x${QUALIFIER} == *debug* ]]; then echo ''; else echo '-O3'; fi )
 
 CXX := $(shell if [[ x${QUALIFIER} == *c7* ]]; then echo 'clang++'; else echo 'g++'; fi )
 
-LDFLAGS := -L${ROOTSYS}/lib -lCore -L${GSL_LIB} -lgsl -lgslcblas
+LDFLAGS := -L${ROOT_LIB} -lCore -lHist -L${GSL_LIB} -lgsl -lgslcblas
```

Comment:
`$ROOTSYS/lib -> $ROOT_LIB` doesn't work on the default gpvm ups build, because apparently that package doesn't define `$ROOT_LIB`. Could you work some more makefile magic to have this work in both cases?